lib/experimental/Layouts/Utils/Tag/index.tsx

```diff
@@ -1,24 +1,21 @@
-import { Avatar } from "@/experimental/Information/Avatar"
+import { UserAvatar } from "@/experimental/Information/Avatars/UserAvatar"
 import { cn } from "@/lib/utils"
 import { ComponentProps, forwardRef } from "react"
-import { getColorFromText } from "../helper"
 
 interface TagProps {
   text: string
-  avatar?: ComponentProps<typeof Avatar>
+  avatar?: {
+    firstName?: string
+    lastName?: string
+    src?: string
+    size?: ComponentProps<typeof UserAvatar>["size"]
```

Comment:
Do we want everything to be optional, or just the `avatar` prop? My guess is something needs to be required here.

Reply:
To be honest, I just adapted it a bit to work, but this Tag is getting deprecated soon, replacing it with #751